pandas/core/indexes/base.py

```diff
@@ -1493,13 +1524,16 @@ def _mpl_repr(self):
     # introspection
     @property
     def is_monotonic(self):
-        """ alias for is_monotonic_increasing (deprecated) """
+        """
+        Alias for is_monotonic_increasing.
+
+        .. deprecated :: """
```

Comment:
This is not actually deprecated (see that the code doesn't raise a deprecation warning.

Can you remove the deprecated directive (also the deprecated in brackets as you did).

@jreback do we want to deprecate this? (that would be for another PR)